Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -70,12 +70,7 @@ public void setDocType(String docType) {
      */
     public String getDocType() {
         if (Objects.isNull(docType)) {
-            String monograph = (String) getAllDoctypes()
-                    .stream()
-                    .filter(typ -> typ.getValue().equals("Monograph"))
-                    .findFirst()
-                    .get().getValue();
-            setDocType(Objects.isNull(monograph) || monograph.isEmpty() ? (String) getAllDoctypes().get(0).getValue() : monograph);
+            setDocType((String) getAllDoctypes().get(0).getValue());
```

Comment:
You need to verify that `getAllDoctypes` does not return an empty list before retrieving the first object from it!